Sources/SwiftGen/Config/Config+Example.swift

```diff
@@ -0,0 +1,81 @@
+//
+// SwiftGen
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+extension Config {
+  static func example(versionForDocLink: String, commentAllLines: Bool = true) -> String {
+    var content = exampleYAMLContent(version: versionForDocLink)
+    if commentAllLines {
+      content = content
+        .split(separator: "\n", omittingEmptySubsequences: false)
+        .map { $0.isEmpty ? "" : $0.hasPrefix("#") ? "#\($0)" : "# \($0)" }
```

Comment:
~What's the purpose of this? Just adding an extra space (or am I missing something)?~
Missed the if test above. Not sure if a comment here would be helpful.

Reply:
Yeah it's so that blank lines are not commented, but lines that are already a comment –starting with just `#` to explain the config content – would be commented with `## Blah` and not `# # Blah`, and only the lines that are not originally commented and are supposed to be actual YAML content be commented with `# `

Reply:
Fixed in latest commit.